lte/gateway/python/magma/pipelined/mobilityd_client.py

```diff
@@ -0,0 +1,60 @@
+"""
+Copyright (c) Facebook, Inc. and its affiliates.
+All rights reserved.
+
+This source code is licensed under the BSD-style license found in the
+LICENSE file in the root directory of this source tree.
+"""
+
+import grpc
+import logging
+
+from magma.common.service_registry import ServiceRegistry
+from orc8r.protos.common_pb2 import Void
+from lte.protos.mobilityd_pb2 import GWInfo
+from lte.protos.mobilityd_pb2_grpc import MobilityServiceStub
+
+SERVICE_NAME = "mobilityd"
+IPV4_ADDR_KEY = "ipv4_addr"
+
+
+def get_mobilityd_gw_info() -> GWInfo:
+    """
+    Make RPC call to 'GetGatewayInfo' method of local mobilityD service
+    """
+    try:
+        chan = ServiceRegistry.get_rpc_channel(SERVICE_NAME,
+                                               ServiceRegistry.LOCAL)
+    except ValueError:
+        logging.error('Cant get RPC channel to %s', SERVICE_NAME)
+        return
+
+    client = MobilityServiceStub(chan)
+    try:
+        return client.GetGatewayInfo(Void())
+    except grpc.RpcError as err:
```

Comment:
You should return an empty GWInfo() here and below or else you break your type annotation contract